python/mkdocs/docs/index.md

```diff
@@ -434,6 +435,14 @@ Table(
 )
 ```
 
+Altering the table:
+
+```python
+table.alter().update_namespace_properties(
+    updates={"department": "finance"}, removes=["owner"]
```

Comment:
In the REST spec, the second argument is `removals`. Should we match that?

Reply:
Ah, that's correct. I went a bit back and forth because of https://github.com/apache/iceberg/pull/6317. The actual code expects `removals`, but I didn't update the docs accordingly. Thanks for spotting 👍🏻